Unable to create test container in dev-env with latest sdk version

Steps to reproduce:
```
@:~/workspace/neofs-dev-env$ make up && make prepare.ir && sudo make prepare.storage
⇒ Download blockchain dump from https://github.com/nspcc-dev/neofs-contract/releases/download/v0.15.5/devenv_mainchain_notary_disabled.gz
...
Adding self-signed tls certs to trusted store
@:~/workspace/neofs-dev-env$ 
@:~/workspace/neofs-dev-env$ 
@:~/workspace/neofs-dev-env$ neofs-cli --rpc-endpoint 192.168.130.71:8080 --wallet /home/annikifa/workspace/neofs-dev-env/wallets/wallet.json container create  --policy "REP 1 IN LOC_MOW SELECT 1 FROM LOC_MOW FILTER 'UN-LOCODE' EQ 'RU MOW' AS LOC_MO" --await --force
Enter password > 
panic: cannot create context from nil parent

goroutine 1 [running]:
context.WithDeadline({0x0?, 0x0?}, {0x146f040?, 0x37e11d600?, 0x146f040?})
	context/context.go:436 +0x41b
context.WithTimeout({0x0, 0x0}, 0x11?)
	context/context.go:507 +0x4a
github.com/nspcc-dev/neofs-api-go/v2/rpc/client.(*Client).openGRPCConn(0xc000366040, {0x0?, 0x0?})
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/client/connect.go:51 +0xca
github.com/nspcc-dev/neofs-api-go/v2/rpc/client.(*Client).createGRPCClient.func1()
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/client/connect.go:18 +0x45
sync.(*Once).doSlow(0x8?, 0xc0001c2090?)
	sync/once.go:68 +0xc2
sync.(*Once).Do(...)
	sync/once.go:59
github.com/nspcc-dev/neofs-api-go/v2/rpc/client.(*Client).createGRPCClient(0x203000?, {0x0?, 0x0?})
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/client/connect.go:17 +0x76
github.com/nspcc-dev/neofs-api-go/v2/rpc/client.(*Client).initGRPC(0xc000366040, {{0xd2e61b, 0x26}, {0xd0d2ca, 0x7}, 0x1}, 0xc00037e280)
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/client/init.go:71 +0x5c
github.com/nspcc-dev/neofs-api-go/v2/rpc/client.(*Client).Init(0x7f8f3b2f9108?, {{0xd2e61b, 0x26}, {0xd0d2ca, 0x7}, 0x1}, {0xc000399388, 0x1, 0x18?})
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/client/init.go:48 +0xfd
github.com/nspcc-dev/neofs-api-go/v2/rpc/client.SendUnary(0x20?, {{0xd2e61b, 0x26}, {0xd0d2ca, 0x7}, 0x1}, {0xe4d898, 0xc000116048}, {0xe4d910, 0xc000116060}, ...)
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/client/flows.go:15 +0x8a
github.com/nspcc-dev/neofs-api-go/v2/rpc.Balance(0xc000348060?, 0x4?, {0xc000399388, 0x1, 0x1})
	github.com/nspcc-dev/neofs-api-go/v2@v2.13.2-0.20221005093543-3a91383f24a9/rpc/accounting.go:23 +0x126
github.com/nspcc-dev/neofs-sdk-go/client.(*Client).Dial(0xc000366000, {{0xc000042258, 0x13}, 0x0, 0x1, 0x37e11d600, 0x1, 0x37e11d600, {0x0, 0x0}})
	github.com/nspcc-dev/neofs-sdk-go@v1.0.0-rc.6.0.20221005093951-1325b4f27218/client/client.go:109 +0x2c5
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client.GetSDKClient(0xc0003333e0, {{0xe591e0?, 0xc000116018?}})
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client/sdk.go:62 +0x24f
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client.getSDKClientByFlag(0x0?, {0xd140f7?, 0x4f?})
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client/sdk.go:38 +0xd5
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client.GetSDKClientByFlag(0x7ffc4f9a4411?, 0x0?, {0xd140f7?, 0x0?})
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client/sdk.go:24 +0x2c
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/container.glob..func1(0x1459760?, {0xd09201?, 0x8?, 0x8?})
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/container/create.go:44 +0x97
github.com/spf13/cobra.(*Command).execute(0x1459760, {0xc0000d9480, 0x8, 0x8})
	github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0x1457460)
	github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.5.0/command.go:918
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules.Execute()
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/root.go:51 +0x25
main.main()
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/main.go:6 +0x17

```

--- a/client/client.go
+++ b/client/client.go
@@ -105,6 +105,10 @@ func (c *Client) Dial(prm PrmDial) error {
 
 	c.setNeoFSAPIServer((*coreServer)(&c.c))
 
+	if prm.parentCtx == nil {
+		prm.parentCtx = context.Background()
+	}
+
 	// TODO: (neofs-api-go#382) perform generic dial stage of the client.Client
 	_, err := rpc.Balance(&c.c, new(v2accounting.BalanceRequest),
 		client.WithContext(prm.parentCtx),

